sdk/cosmos/azure-cosmos/README.md

```diff
@@ -269,28 +258,44 @@ We have added the ability to modify the default Direct TCP options utilized by t
 
 Values that are in error are ignored.
 
-### Common Perf Tips
-
-There is a set of common perf tips written for our Java SDK. It is available [here](https://docs.microsoft.com/en-us/azure/cosmos-db/performance-tips-async-java).
-
-## FAQ
-
-We have a frequently asked questions which is maintained [here](faq/).
-
-## Release changes
-
-Release changelog is available [here](changelog/).
-
-## Contribution and Feedback
-
-This is an open source project and we welcome contributions. If you would like to become an active contributor to this project please follow the instructions provided in [Azure Projects Contribution Guidelines](http://azure.github.io/guidelines/). Instructions on how to fetch and build the code can be found in [dev.md](./dev.md). Our PRs have CI that will run after a contributor has reviewed your code. You can run those same tests locally via the instructions in [dev.md](./dev.md). 
-
-If you encounter any bugs with the SDK please file an [issue](https://github.com/Azure/azure-cosmosdb-java/issues) in the Issues section of the project.
-
-## License
-
-[MIT License](LICENSE)
-
-Copyright (c) 2018 Copyright (c) Microsoft Corporation
+## Next Steps
+
+- Samples are explained in detail [here][samples_readme]
+- Go through [quickstart][quickstart] - Building a java app to manage CosmosDB SQL API data
+- [Read more about Azure CosmosDB Service][cosmos_docs]
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a 
+[Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights 
+to use your contribution.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate 
+the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to 
+do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] 
+or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+
+<!-- LINKS -->
+[source_code]: src
+[cosmos_introduction]: https://docs.microsoft.com/en-us/azure/cosmos-db/
+[api_documentation]: https://azuresdkdocs.blob.core.windows.net/$web/java/azure-cosmos/4.0.1-beta.3/index.html
```

Comment:
end users may search for the word "javadoc" can we add that as a keyword as well?

Reply:
Yes, will do.

Reply:
Done.